src/comm/UDPLink.cc

```diff
@@ -317,32 +312,43 @@ void UDPConfiguration::addHost(const QString& host)
             for (int i = 0; i < hostAddresses.size(); i++)
             {
                 // Exclude loopback IPv4 and all IPv6 addresses
-                if (!hostAddresses.at(i).toString().contains(":"))
+                if (!hostAddresses.at(i).toString().contains(":") && !!hostAddresses.at(i).toString().startsWith("127"))
```

Comment:
I'm not too sure about this one. We have quite often running the autopilot code and the GCS on the same machine. Could you think of any reason to disallow localhost here?